### PR TITLE
Changed Jason to Andre per their request

### PR DESCRIPTION
**Pull request by ianbjacobs**
(Sorry for other text which has not changed but maybe a tidy issue).

### CODE REVIEW

**Comment by ianbjacobs** [comment]
@adrianhopebailie, changed editor from Shopify per their request.

File: index.html [hunk 1]
```diff
@@ -1617,11 +1617,11 @@
           </h2>
           <p>
             This attribute indicates the total amount being requested for
-            payment. It is of type <a>PaymentCurrencyAmount</a> dictionary as defined in
-            [[payment-request]], and initialized with a <a>structured clone</a>
-            of the <a>total</a> field of the <a>PaymentDetailsInit</a> provided
-            when the corresponding <a>PaymentRequest</a> object was
-            instantiated.
+            payment. It is of type <a>PaymentCurrencyAmount</a> dictionary as
```
Comment (marcoscaceres) [inline index.html]: Any idea what these changes here are?

**Comment by marcoscaceres** [comment]
Note: please be sure to prefix the commit message with "chore:", so we can distinguish it from substantive changes (helps a lot later, when we need to know what's changed from one step in the W3c process to the next).

**Comment by marcoscaceres** [comment]
To fix:
git commit --amend
git push -f

File: index.html (hunk 1 above)
Comment (ianbjacobs) [reply index.html]: It looks like a "failure to tidy something" error. Otherwise the text has not changed.

Comment (marcoscaceres) [reply index.html]: Odd... Thanks for checking.

**Review by adrianhopebailie — APPROVED** [review]
Fixed the commit after merging in another HTML fix from @romandev

**Pull request merged**